packages/runtime/src/plugins/utils/element.ts

```diff
@@ -0,0 +1,61 @@
+import { Node, Element } from 'slate'
```

Comment:
FWIW I hate util folders. I don't think this is hot stuff. I don't like having `ElementUtils`. 

It's a temporary thing until I find best practices on this stuff.

Reply:
Slate seems to call them[ [custom helpers](https://kevinzakka.github.io/robopianist-demo/) so maybe we can just go for what they go for? Maybe look at the Slate source code to see how they organize their built-in helpers?

Reply:
Seems like you shared the wrong link. Unless I am missing something :P 

Yeah, I have seen those. I didn't like the idea of wrapping the standard interfaces initially because there are already enough import options named Element, Editor, Range, etc. Now that I am thinking about it though I like the idea of having MakeswiftElement. It would clear things up so that you don't have to check the import statements as much. 

For another day regardless!